src/Blocks/components/load-more/docs/story.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```

Comment:
please fix this story as I see in the code you dont have nothing similar to this